src/SOS/lldbplugin/soscommand.cpp

```diff
@@ -177,7 +177,6 @@ sosCommandInitialize(lldb::SBDebugger debugger)
     g_services->AddCommand("dumpmodule", new sosCommand("DumpModule"), "Displays information about a EE module structure at the specified address.");
     g_services->AddCommand("dumpmt", new sosCommand("DumpMT"), "Displays information about a method table at the specified address.");
     g_services->AddCommand("dumpobj", new sosCommand("DumpObj"), "Displays info about an object at the specified address.");
-    g_services->AddCommand("dumpruntimetypes", new sosCommand("DumpRuntimeTypes"), "Finds all System.RuntimeType objects in the GC heap and prints the type name and MethodTable they refer too.");
```

Comment:
This should be changed to g_service->AddManagedCommand("dumpruntimetypes", "Finds ....  See the dumpasync command for an example.

This creates a direct lldb `dumpruntimetypes` that can be run with the sos prefix. 

If you haven't done this for your other new managed commands they needed to be added here. I forgot to mention this in previous PRs. It does look like dumpheap, eeheap, gcheapstat, gcroot, gcwhere, etc. all have to be added.  I actually have these changes in one of my branches here: https://github.com/mikem8361/diagnostics/blob/b0471f94c48f8c17903e3e428e00a2d05792ed0d/src/SOS/lldbplugin/soscommand.cpp#L175

Reply:
Oh interesting!  I will fix the PR for just these commands, I'll submit a second PR for the other commands I've changed.